src/components/Input.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { memo } from "react";
+
+function Input({ job, setJob }) {
```

Comment:
```suggestion
function TextInput({ job, setJob }) {
```

Please do not use overly generic names.